operations/business-operations/customer-support.md

```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
 - Customer Satisfaction.  
 
 Details of these metrics for our Enterprise customers can be found [here](https://handbook.mattermost.com/operations/business-operations/analytics/metrics-definitions#support-tickets).
+
+## Sales Support
+
+Support is also provided for customers using our self-service portal which can be access [here](https://customers.mattermost.com/signup). Customer requesting assistance from here will be directed to [this](https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000852671) page.
+
+## Ticket workflow (Sales Support)
+
+The Sales Support Team use a slightly difernet workflow to the Technical Support Team and this can be seen here:
+
+![](../../.gitbook/assets/Sales_Support_Process_Workflow.png)
```

Comment:
This image doesn't show up. Has this been uploaded?

Reply:
THat's odd, it was showing in preview mode for me yesterday, the file is here:

https://github.com/mattermost/mattermost-handbook/blob/0.2.1/.gitbook/assets/Sales_Support_Process_Workflow.png